Monika After Story/game/script-topics.rpy

```diff
@@ -15151,6 +15151,46 @@ label monika_songwriting:
     m 1eua "I'd love to sing with you [_if] I come to your world, [player]."
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_physicalmedia",
+            category=['media'],
+            prompt="Physical media",
+            random=True
+        )
+    )
+
+label topic_physical_media:
+
+    m 1eud "You know, with technology going so far in your world, the way data is stored have changed drastically over the last few decades."
+    m 1eua "In the '80s, Magnetic cassette tapes and floppy disks were the norm, before in 1986 the compact disks that can store various data came out."
```

Comment:
`    m 1eua "In the '80s, magnetic cassette tapes and floppy disks were the norm, before in 1986 the compact disks that can store various data came out."`